bokehjs/src/coffee/common/plot.coffee

```diff
@@ -122,17 +122,17 @@ class PlotView extends ContinuumView
   
   init_webgl: () ->
 
-    # Get visible and invisible canvas 
-    # (doing it like this makes it easy to swap them during debugging)    
-    glcanvas = document.createElement('canvas')
-    
-    # Create GL context to draw to
+    # We use a global invisible canvas and gl context. By having a global context,
+    # we avoid the limitation of max 16 contexts that most browsers have. 
+    glcanvas = window._bokeh_gl_canvas
```

Comment:
Does this have to be set on `window`? Maybe you could use a module-level variable?